json_annotation/lib/src/json_serializable.g.dart

```diff
@@ -55,8 +58,9 @@ JsonSerializable _$JsonSerializableFromJson(Map<String, dynamic> json) =>
       },
       fieldKeyMap: const {
         'anyMap': 'any_map',
-        'createFieldMap': 'create_field_map',
         'createFactory': 'create_factory',
+        'createFieldMap': 'create_field_map',
```

Comment:
revert the sort here. I know it's not ordered, but it keeps the diff readable

Reply:
done 👍